### PR TITLE
Improve FileSharing and Notes headers

### PR DESCRIPTION
**Pull request by Paiusco**
- Remove deprecated TrebleShot app
- Add SnapShot for new FileSharing option
- Add Joplin app on Notes section, even though it can count as ToDos as well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ Table of Contents
 
 
 ### • File Sharing
-- [**TrebleShot**](https://github.com/trebleshot/android) <sup>**[[F-Droid](https://f-droid.org/app/com.genonbeta.TrebleShot)]**</sup>
+- [**SnapShot**](https://github.com/fm-sys/snapdrop-android) <sup>**[[F-Droid](https://f-droid.org/packages/com.fmsys.snapdrop/)]**</sup>
```
Comment (offa) [inline README.md:181]:
```suggestion
- [**SnapShot**](https://github.com/fm-sys/snapdrop-android) <sup>**[[F-Droid](https://f-droid.org/app/com.fmsys.snapdrop)]**</sup>
```

File: README.md [hunk 2]
```diff
@@ -298,6 +298,7 @@ Table of Contents
 
 
 ### • Notes
+- [**Joplin**](https://github.com/laurent22/joplin/)
```
Comment (offa) [inline README.md:301]: Really, really minor:

```suggestion
- [**Joplin**](https://github.com/laurent22/joplin)
```

**Comment by offa** [comment]
I've added some minor editorial notes, but otherwise ok.
Thank you! :+1:

**Comment by Paiusco** [comment]
> I've added some minor editorial notes, but otherwise ok. Thank you! 👍

Both fixed!

**Comment by offa** [comment]
Perfect!

**Pull request merged**